frontend/public/components/utils/service-level.tsx

```diff
@@ -48,7 +48,7 @@ const TrialDaysLeft: React.FC<{
 
   let variant: 'warning' | 'danger' = 'warning';
 
-  let alertText = t('public~{{count}} day remaining', { count: trialDaysLeft });
+  let alertText = t('public~{{count}} days remaining', { count: trialDaysLeft });
```

Comment:
what happens if `trialDaysLeft === 1`?

Reply:
@krishagarwal278 Did you verify this fix? This would not fix the issue. This issue is happening due to the limitation of i18n pluralization script, which does handle plural strings before the last strings in phrases and sentences. It is a known issue.  Usually we update the impacted strings manually in the “locales” files. 
Update the string here: ["locales/en/public.json"](https://github.com/openshift/console/blob/main/frontend/public/locales/en/public.json#L1618)

However it would be good to  revisit the issue if anyone is interested in finding a better solution.